README.md

```diff
@@ -1,33 +1,64 @@
-@techchange/slack-webhooks
+console-to-slack
 ==========================
 
-Private NPM module for integrating with Slack webhooks. Currently supports overriding console.error and sending error messages to Slack. 
+NPM module for integrating with Slack webhooks. Currently supports overriding console.log, console.warn, and console.error to send messages to Slack.
 
 ## Installation
 ```
-npm install @techchange/slack-webhooks
+npm install console-to-slack
 ```
 
 ## Usage
 Include this module:
 ```javascript
-const tcSlackWebhooks = require('@techchange/slack-webhooks')(settings);
+const consoleToSlack = require('console-to-slack')(options);
 ```
 
-#### Settings
-For slack webhooks to work, settings must include:
+You can use the following methods:
 
-- **settings.slackWebhookUrl**: The url of the slack thread to send messages to.
-- **settings.name**: The name of the service. This is necessary to distinguish the location of the Slack message.
+- **consoleToSlack.overrideAll**: Overrides console.log, console.warn, and console.error to send messages to Slack via webhooks.
+- **consoleToSlack.overrideConsoleLog**: Overrides console.log to send error messages to Slack via webhooks.
```

Comment:
What about naming these methods `overrideLog`, `overrideWarn`, `overrideError`? I don't think that it loses any semantic meaning and is shorter.